src/protocols/bdx/DiagnosticLogs.h

```diff
@@ -18,6 +18,8 @@
 
 #pragma once
 
+#include <stdint.h>
```

Comment:
Since the project is C++ based it's better to use <c...> versions of includes (if available).